<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDoubleRangeSpinEdit</class>
 <widget class="QWidget" name="CDoubleRangeSpinEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>20</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="FromLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>from</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="ValueFromSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="SuffixLabel1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ToLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="ValueToSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="SuffixLabel2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ValueFromSpinBox</tabstop>
  <tabstop>ValueToSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
